service/src/main/java/uk/gov/hmcts/reform/fpl/model/Judge.java

```diff
@@ -34,4 +38,11 @@ public String getJudgeName() {
         }
         return judgeLastName;
     }
+
+    public boolean hasEqualJudgeFields(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
+        return Objects.equals(getJudgeTitle(), judgeAndLegalAdvisor.getJudgeTitle())
+            && StringUtils.equals(getOtherTitle(), judgeAndLegalAdvisor.getOtherTitle())
+            && StringUtils.equals(getJudgeLastName(), judgeAndLegalAdvisor.getJudgeLastName())
+            && StringUtils.equals(getJudgeFullName(), judgeAndLegalAdvisor.getJudgeFullName());
```

Comment:
minor: but we might want to prefer `getJudgeLastName().equals(judgeAndLegalAdvisor.getJudgeLastName())`

Reply:
If the field is null then this will fail (and it is possible/expected) :( Hence why I took the other approach